common/cme_api_postman/v1/cme_api_v1.postman_collection.json

```diff
@@ -1,1235 +1,1241 @@
+                            }, {
+                                "key": "X-chkp-sid",
+                                "value": "{{session}}"
+                            }
+                        ],
+                        "body": {
+                            "mode": "raw" ,
+                            "raw": "{\"name\" : \"<account_name>\", \"regions\" : [\"eu-west-1\", \"us-east-2\"], \"credentials_file\" : \"AWS_credentials_file_name\", \"access_key\" : \"<access_key>\", \"secret_key\" : \"<secret_key>\", \"sts_role\" : \"<sts_role>\", \"sts_external_id\" : \"<sts_external_id>\", \"scan_gateways\" : \"<scan_gateways>\", \"scan_vpn\" : \"<scan_vpn>\", \"scan_load_balancers\" : \"<scan_load_balancers>\", \"scan_subnets\" : \"<scan_subnets>\", \"communities\" : [\"< community_a >\", \"< community_b > \"], \"deletion_tolerance\" : \"<deletion_tolerance>\", \"sub_accounts\" : [{\"name\" : \"sub_account_a\",\"credentials_file\" : \"AWS_credentials_file_name\"}, {\"name\" : \"sub_account_b\", \"access_key\" : \"<access_key>\", \"secret_key\" : \"<secret_key>\"}, {\"name\" : \"sub_account_c\", \"sts_role\" : \"<sts_role>\", \"sts_external_id\" : \"<sts_external_id>\"}]}",
+                            "options": {
+                                "raw": {
+                                    "language": "json"
+                                }
+                            }
+                        },
+                        "url": {
+                            "raw": "{{server}}/v1.8/cme-api/v1/accounts/aws",
+                            "host": [
+                                "{{server}}"
+                            ],
+                            "path": [
+                                "v1.8",
+                                "cme-api",
+                                "v1",
+                                "accounts",
+                                "aws"
+                            ]
+                        }
+                    },
+                    "response": []
+                }, 
 				{
-					"name": "cme_api - get cme version",
-					"request": {
-						"method": "GET",
-						"header": [
+                    "name": "PUT set AWS account",
+                    "request": {
+                        "method": "PUT",
+                        "header": [
 							{
-								"key": "Content-Type",
-								"value": "application/json"
-							},
+                                "key": "Content-Type",
+                                "value": "application/json"
+                            }, {
+                                "key": "X-chkp-sid",
+                                "value": "{{session}}"
+                            }
+                        ],
+                        "body": {
+                            "mode": "raw",
+                            "raw": "{\"regions\" : [\"eu-west-1\", \"us-east-2\"], \"credentials_file\" : \"AWS_credentials_file_name\"}",
```

Comment:
Please update to: "raw": "{\"regions\": [\"eu-west-1\", \"us-east-2\"], \"credentials_file\": \"AWS_credentials_file_name\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\", \"sts_role\": \"<sts_role>\", \"sta_external_id\": \"<sts_external_id>\", \"scan_gateways\": \"<scan_gateways>\", \"scan_vpn\": \"<scan_vpn>\", \"scan_load_balancers\": \"<scan_load_balancers>\", \"scan_subnets\": \"<scan_subnets>\", \"communities\": [\"<community_a>\", \"<community_b>\"], \"deletion_tolerance\": \"<deletion_tolerance>\", \"sub_accounts\": [{\"name\": \"sub_account_a\",\"credentials_file\": \"AWS_credentials_file_name\"}, {\"name\": \"sub_account_b\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\"}, {\"name\": \"sub_account_c\", \"sts_role\": \"<sts_role>\", \"sts_external_id\": \"<sts_external_id>\"}]}",